x/tokenomics/keeper/settle_session_accounting.go

```diff
@@ -29,40 +29,40 @@ func (k Keeper) SettleSessionAccounting(
 ) error {
 	logger := k.Logger().With("method", "SettleSessionAccounting")
 
-	settlementAmt := sdk.NewCoin("upokt", math.NewInt(0))
+	settlementCoin := cosmostypes.NewCoin("upokt", math.NewInt(0))
```

Comment:
Thoughts on pluralizing this to `Coins`?

Reply:
`settlementCoins` is used elsewhere and is a `[]cosmostypes.Coin` type.